frontend/src/metabase/public/components/EmbedFrame.jsx

```diff
@@ -78,7 +80,8 @@ class EmbedFrame extends Component {
                 <div className="flex">
                   <SyncedParametersList
                     className="mt1"
-                    dashboard={this.props.dashboard}
+                    question={question}
+                    dashboard={dashboard}
```

Comment:
Changes to `EmbedFrame` will most likely result in conflicts with `master` after #27746. Probably it'd make sense to sync the base branch and this branch with the latest `master`